examples/plot_anomaly_bench.py

```diff
@@ -0,0 +1,118 @@
+"""
+==================================
+Anomaly detection's ROC benchmarks
+==================================
+
+This example benchmarks outlier detection algorithms using ROC curves.
+The data are from classical anomaly detection datasets, and the algorithms'
+performance is assessed in an outlier detection context:
+
+1. The algorithms are trained on the whole dataset which is assumed to contain
+outliers.
+2. The ROC curve is computed on the same dataset using the knowledge of the
+labels.
+
+This experiment uses area under the ROC curve score (AUC) to compare
```

Comment:
I would remove this sentence as it is already said above that we are using ROC curves.